<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btn_file">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>331</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_calc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bubble">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bubble</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_scatter_matrix">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scatter matrix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_par_coord">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Parallel coordinates</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
